bitnami/kafka/README.md

```diff
@@ -208,6 +208,9 @@ The configuration can easily be setup with the Bitnami Apache Kafka Docker image
 * `KAFKA_CFG_NODE_ID`: Unique node id, required when running in KRaft mode. No defaults.
 * `KAFKA_CFG_CONTROLLER_QUORUM_VOTERS`: Map of id/endpoint information for the set of controller quorum voters in a comma-separated list of {id}@{host}:{port} entries. No defaults.
 * `KAFKA_RAFT_CLUSTER_ID`: Kafka cluster ID when using Kafka Raft (KRaft). No defaults.
+* `KAFKA_CFG_OFFSETS_TOPIC_REPLICATION_FACTOR`: specifies the replication factor for the `__consumer_offsets` topic. Default: **1**.
+* `KAFKA_CFG_TRANSACTION_STATE_LOG_REPLICATION_FACTOR`: specifies the replication factor for the `__transaction_state` topic. Default: **1**.
+* `KAFKA_CFG_TRANSACTION_STATE_LOG_MIN_ISR`: overrides the `min.insync.replicas` configuration for the `__transaction_state` topic. Default: **1**.
```

Comment:
Is this necessary to document? What's wrong with the section below? 

> Additionally, any environment variable beginning with `KAFKA_CFG_` will be mapped to its corresponding Apache Kafka key

Reply:
@OneCricketeer Without these configs you can't have a genuine HA Cluster, it is time-consuming for new people, and I thought it would be good to document them.

Reply:
I understand the purpose of them, but I feel like it should be an exercise of the reader to interpret the mentioned prefix and apply any additional config they may need, by following the Kafka documentation

Reply:
I do agree with @OneCricketeer. I see your point here, but most of `KAFKA_CFG` entries in the documentation are added to further bring some remark on their usage to light. I think that the note:

> Additionally, any environment variable beginning with KAFKA_CFG_ will be mapped to its corresponding Apache Kafka key

is precise and adding these three entries do not add much value (and in turn makes the section even lengthier).

Reply:
Please, remove this from here.

Reply:
Please, remember to remove this lines